app/build.gradle

```diff
@@ -29,12 +29,12 @@ android {
             shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
 
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://www.eventyay.com/api/v1/"'
-            resValue "string", "default_base_url", '"https://www.eventyay.com/api/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            resValue "string", "default_base_url", '"https://api.eventyay.com/v1/"'
         }
         debug {
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-dev.herokuapp.com/api/v1/"'
-            resValue "string", "default_base_url", '"https://open-event-dev.herokuapp.com/api/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api.herokuapp.com/v1/"'
+            resValue "string", "default_base_url", '"https://open-event-api.herokuapp.com/v1/"'
```

Comment:
I think this should remain so, so that user is aware in what form the base url is. Furthermore, not relevant to this issue